bounties/level-1/basic-account-viewer.md

```diff
@@ -0,0 +1,51 @@
+# Basic account viewer
+
+[🔵 Start hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.md&title=%F0%9F%94%B5+%60basic-account-viewer%60) this bounty
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 200 XLM | 1 Credits | Continuos
```

Comment:
I would like the prize level to be 800 XLM and the review to be 2 Credits. This is a good bounty and I would like to see a lot of folks tackle this. Better to start too high and scale it back over time than too low and miss out on a great opportunity to attract interest.